packages/rancher-logging/overlay/templates/outputs/elasticsearch/secret.yaml

```diff
@@ -1,14 +0,0 @@
-{{- if .Values.elasticsearch.enabled }}
```

Comment:
Are we just going to document how users should set these secrets up ? We could deploy an empty secret but I'm guessing by then its too late.

Reply:
The ui is supposed to create a secret for them, we will need to document it for cli users.

Reply:
Do we know the default secret name ? Should put that in the default values ? Or not because that would assume user has authentication turned on at ES side.

Reply:
I am in favor of not defaulting a name because it isn't normally required.